internals/overlord/state/notices.go

```diff
@@ -192,6 +228,13 @@ type AddNoticeOptions struct {
 	Time time.Time
 }
 
+func optionIntToString(maybeInt *int) string {
+	if maybeInt == nil {
+		return `"null"`
```

Comment:
I think we should use `"<nil>"` instead (as Go's own fmt does for nil pointers).